### DISCUSSION TITLE
Wondering the difference between CCx and CCx0 or Crop_CCx

### DISCUSSION THREAD

**Discussion by shqwu** [aquacropos/aquacrop#64] [General]
Hi Tom,

I noticed that the equation to calculate declining CC is slightly different from the Aquacrop manual. You introduced CCx0 in your script of defining CC development but I couldn't found it in the manual. But when I tried to use the equation from the manual to calculate declining CC, it decline sharply instead of gradually, which I think is the reason you modify the equation. Just wondering where you get the CCx0 and how you modify the declining CC equation.

Thank you so much!

**Comment by shqwu** [comment]
Hi Tom,

I think I know why you introduce CCx0 in the cc_development since you update CCx by CCx0 when calculating canopy decline. But I am wondering if you notice that because of the exponential relationship, it's impossible to reach either CCx or CCx*0.98 within the given days/GDDs but extremely close to it.

**Reply by pacs27** [reply]
Hi @shqwu,

CCx0 is only used in the decrease of the canopy cover. In the official repository in fortran the cc in this situation is calculated like this:

https://github.com/KUL-RSDA/AquaCrop/blob/b36e30745c08a848113b84e7a6454eda8784b7e9/src/simul.f90#L4976-L4982

So CCX0 is used to calculate the CDCadjusted which is calculated in this equation:

https://github.com/KUL-RSDA/AquaCrop/blob/b36e30745c08a848113b84e7a6454eda8784b7e9/src/simul.f90#LL420C5-L420C62

CCx is in fact CCxadjusted and CCx0 is CCx
